### ISSUE TITLE
Settings panes need additional ARIA

### ISSUE THREAD

**Issue by becka11y** [readium/readium-js-viewer#145]
On the settings dialog the Settings and Keyboard Shortcuts tabs are labeled as buttons. Ideally these should be identified as tabs via role=tab and should have an aria-controls associated with the panel. The containing element for the tabs should be marked as tablist. Each associated panel should have the tabpanel role.  Implementing tab, tablist and tabpanel roles also implies arrow key access to the tabs in the tab list. 

Marking these elements as a tab panel may be a stretch.  For now they could be left as buttons with the additional attributes aria-expanded=true/false  and aria-contols=id of associated tab panel. This would make it clearer to the screen reader user that this button hides/shows additional content. 

Tested with VoiceOver on OS X 10.9.4 with Chrome Extension readium-js-viewer sha = eedd52eb (Tue, Aug 5, 2014)
